cypress/integration/sema4_challenge/google_checkout.spec.js

```diff
@@ -0,0 +1,44 @@
+/// <reference types="Cypress" />
+
```

Comment:
You have the right idea here - abstracting individual steps into separate functions can make these flows easier to read.  I'm curious: why not just create all of these as Cypress commands? (You don't have to use the `Commands` API; I'm just interested in your train of thought here)